src/components/Dialog/index.tsx

```diff
@@ -0,0 +1,12 @@
+import { DialogComponent } from "./Dialog";
```

Comment:
This file doesn't need to be a .tsx file since there is no TSX code being used in it. Instead, you can use .ts